repository/flink/operator/operator.yaml

```diff
@@ -1,5 +1,5 @@
 name: "flink"
-version: "0.1.0"
+version: "0.1.1"
 kudoVersion: 0.3.0
 kubernetesVersion: 1.15.0
```

Comment:
Should we bump this to `1.16.0` as well? Not sure if this will work with `1.15.0`

Reply:
I tested it with v1.15.5

Reply:
the kudo commitment is for 1.16 based on other factors... if it work tho... great!!